test/integration/targets/special_vars/tasks/main.yml

```diff
@@ -56,3 +56,8 @@
   assert:
     that:
       - "(ansible_play_role_names + ansible_dependent_role_names)|unique|sort|list == ansible_role_names|sort|list"
+
+- name: test ansible_parent_role_name(s) functionality by including another role
+  include_role:
+    name: include_parent_role_vars
+    tasks_from: included_by_other_role.yml
```

Comment:
This test doesn't appear to have any relation to the `special_vars` integration test. It should probably be its own test target instead.

Reply:
The `ansible_parent_role_*` vars are special vars, as such, have a direct relation to being tested along with other special vars. However, since these special vars are only activated by including another role, I opted for including said other role from within the `special_vars` integration test.